src/cascadia/ShellExtension/OpenTerminalHere.cpp

```diff
@@ -44,6 +42,60 @@ bool IsDevBuild()
     return isDevBuild;
 }
 
+// Function Description:
+// - Helper function for getting the path to the appropriate executable to use
+//   for this instance of the shell extension. If we're running the dev build,
+//   it should be a `wtd.exe`, but if we're preview or release, we want to make
+//   sure to get the correct `wt.exe` that corresponds to _us_.
+// - If we're unpackaged, this needs to get us `windowsterminal.exe`, because
+//   the `wt*exe` alias won't have been installed for this install.
+// Arguments:
+// - <none>
+// Return Value:
+// - the full path to the exe, one of `wt.exe`, `wtd.exe`, or `windowsterminal.exe`.
+std::wstring _getExePath()

```

Comment:
throw this behind another magic staticizer so we don't do the math every time someone requests it :smile: